nav2_smac_planner/src/a_star.cpp

```diff
@@ -242,15 +290,26 @@ bool AStarAlgorithm<NodeT>::areInputsValid()
   }
 
   // Check if points were filled in
-  if (!_start || !_goal) {
+  if (!_start || _goalsSet.empty()) {
     throw std::runtime_error("Failed to compute path, no valid start or goal given.");
   }
-
```

Comment:
readd the line for separating the code blocks

Reply:
done in https://github.com/ros-navigation/navigation2/pull/4127/commits/3acc50e50f04ead5919f556af3a004221e755ab0